tests/testthat/test-oc_get.R

```diff
@@ -28,12 +28,13 @@ test_that("oc_get returns a response object for Namibia NA countrycode", {
         query_par = list(
           placename = "Windhoek",
           key = Sys.getenv("OPENCAGE_KEY"),
-          countrycode = "NA"
+          countrycode = "NA",
+          limit = 2
         ),
         endpoint = "json"
       )
     ),
-    "HttpResponse"
+    "httr2_response"
```

Comment:
these tests are actually failing with "HTTP 400 Bad Request". I'm not sure why as the same `oc_forward()` query works. I see the first test in this file, that doesn't fail, uses a key for always getting 200. I'm a bit lost.